tests/bson/bson-packedarray-fromJSON_error-001.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+MongoDB\BSON\PackedArray::fromJSON(): invalid JSON
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+echo throws(function() {
+    MongoDB\BSON\PackedArray::fromJSON('foo');
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), PHP_EOL;
+
+echo throws(function() {
+    MongoDB\BSON\PackedArray::fromJSON('{ "foo": "bar" }');
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), PHP_EOL;
+
+echo throws(function() {
+    MongoDB\BSON\PackedArray::fromJSON('{ "00": "bar", "1": "bar" }');
```

Comment:
Is there any need for a test where the invalid key is _not_ the first? Perhaps not, as the extra validation in PHPC means we're no longer relying on libbson's lazy check.

Reply:
Not necessarily, but I'll add one for the sake of completeness.